src/main/java/org/jabref/gui/JabRefDialogService.java

```diff
@@ -148,12 +150,14 @@ public static String shortenDialogMessage(String dialogMessage) {
     @Override
     public void showInformationDialogAndWait(String title, String content) {
         FXDialog alert = createDialog(AlertType.INFORMATION, title, content);
+        Globals.getThemeLoader().installCss(alert.getDialogPane().getScene(), Globals.prefs);
```

Comment:
Does it work, if you move this line inside the createDialog? Then you could reduce some code duplication

Reply:
Yes! didn't really pay attention.